tool/grind/double_check.dart

```diff
@@ -36,12 +36,15 @@ Future<void> doubleCheckBeforeRelease() async {
       ".",
       ...Directory("pkg").listSync().map((entry) => entry.path)
     ]) {
-      var pubspec = Pubspec.parse(File("$dir/pubspec.yaml").readAsStringSync(),
-          sourceUrl: p.toUri("$dir/pubspec.yaml"));
-
-      var package = await client.packageInfo(pubspec.name);
-      if (pubspec.version == package.latestPubspec.version) {
-        fail("${pubspec.name} ${pubspec.version} has already been released!");
+      var pubspecFile = File("$dir/pubspec.yaml");
+      if (pubspecFile.existsSync()) {
```

Comment:
Nit: avoid excess indentation via
```suggestion
      if (!pubspecFile.existsSync()) continue;
```